modules/system/assets/js/framework.js

```diff
@@ -91,7 +91,7 @@ if (window.jQuery.request !== undefined) {
         }
 
         if (useFiles) {
-            requestData = new FormData($form.length ? $form.get(0) : null)
```

Comment:
@Air-Petr I think you could just change it to undefined, null was removed as a supported constructor value.
```suggestion
            requestData = new FormData($form.length ? $form.get(0) : undefined)
```

Reply:
Yes, that works too. Nice variant 👍